testing/benchmark/main.tf

```diff
@@ -9,23 +9,22 @@ terraform {
       source  = "hashicorp/aws"
       version = ">=4.17.1"
     }
+    time = {
+      source  = "hashicorp/time"
+      version = ">=0.9.1"
+    }
   }
 }
 
+resource "time_static" "created_date" {}
+
 locals {
   ci_tags = {
-    environment  = coalesce(var.ENVIRONMENT, "dev")
-    repo         = coalesce(var.REPO, "apm-server")
+    environment  = var.ENVIRONMENT
+    repo         = var.REPO
```

Comment:
repo is basically a duplicate of the team tag in the case of apm-server. Is it worth adding it ?